solidity/ecdsa/.eslintrc

```diff
@@ -0,0 +1,4 @@
+{
+  "root": true,
+  "extends": ["@thesis-co"]
+}
```

Comment:
Don't we want to include the rest of the config as difined in /random-beacon to stay consistent?

Reply:
The thing is I'm not sure which of those entries we do really need. I wanted to go with a minimum config and enhance it as we go. I don't want to end up us copying the config from project to project that comes out unnecessary.

Reply:
Here 3d29977cec02bf47f01a3de6c9ae7db6a8f54cef I added one entry that we need.